mod/01_core/spec/set/all/templating_spec.rb

```diff
@@ -69,7 +70,9 @@
     end
     
     it "should apply to new cards" do
-      expect(Card.new(:name=>"Pete+birthday").content).to eq('Today!')
+      pb = Card.new :name=>"Pete+birthday"
+      expect(pb.raw_content).to eq('Today!')
```

Comment:
the above line fails without the change. "raw_content" is what most views are supposed to use.  (you could easily make the case that #raw_content and #content should be switched, or that we need deeper changes to content handling, but that is for another day ;)